registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/main/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessor.java

```diff
@@ -116,6 +116,9 @@
 
 	@Autowired
 	private Environment env;
+	
+	@Autowired
+	private BioDedupeStage biodedupestage;
```

Comment:
from BioDedupeStage we would call BioDedupeProcessor and from here you are again calling stage.
Will it not create circular dependency? did u test it.